environments/production/migrations/0009-move-to-3-jammy-nodes/couchdb-step-1.yml

```diff
@@ -0,0 +1,2 @@
+target_allocation:
```

Comment:
Curious what this file is used for?

Reply:
In https://confluence.dimagi.com/display/saas/Replace+a+Couch+node+in+Couch+cluster, it's the PLAN_FILE that is used in phase 2.

Reply:
Taking a stab at answering this cause why not 😅 I believe we reference this migration file when running `cchq ${ENV} migrate-couchdb ${PLAN_FILE} plan` in phase 2 of adding this node. And then it looks like that file is used to [allocate shards properly](https://github.com/dimagi/commcare-cloud/blob/2bb8d4bdd71ad8efc3c150c8fc03eeb5591ec47d/src/commcare_cloud/commands/migrations/couchdb.py#L321-L323) via couchdb-cluster-admin.

So then the syntax here is saying that we have three nodes and should make three copies of data across these nodes, which might be the same as saying 1 copy per node?

Reply:
Yeah but you said it better, Graham :)

Reply:
Thanks @dannyroberts and @gherceg. That was super helpful :)